### ISSUE TITLE
Feature request: BetterBibTeX citekey in rename format

### ISSUE THREAD

**Issue by lyndondrake** [jlegewie/zotfile#468]
Hi,

I'd find it very helpful if I could use the BetterBibTeX plugin's citekey in the rename format for my PDFs, so that the PDF filename begins with the citekey.

Is that possible to add?

Best,
Lyndon

**Comment by lyndondrake** [comment]
Actually I've just found earlier issues on this - and `%b` works. But it's not in the documentation yet. If you get a moment to add to the documentation that would be cool. And very much appreciate that it works in the current version!

**Comment by notuntoward** [comment]
I was just looking for this feature.  Would be nice to see it in the docs, so I can figure out how to use it.

**Comment by tjallemaal** [comment]
where do i have to insert %b ???
ive put it to wildcard.user is that correct? because it doesnt work for me

**Comment by lyndondrake** [comment]
@tjallemaal this issue is referring to rename rules in ZotFile (in the Preferences pane for the addin)

**Comment by tjallemaal** [comment]
i didn't know zotfile has its own preferences ... ok, so i insert the {%b}
in both " format for all item types except patents" and "format for patents"?

![Screenshot 2021-01-12 at 20 47 06](https://user-images.githubusercontent.com/69585777/104364947-a3769b00-5517-11eb-84db-cbcd5d4292ac.png)

**Comment by qqobb** [comment]
> ive put it to wildcard.user is that correct?

No, `%b` is already defined in `wildcards.default`. You shouldn't edit `wildcards.user` for the citekey wildcard.

> insert the {%b} in both " format for all item types except patents" and "format for patents"

Yes, this should work.

For troubleshooting ZotFile's file renaming function using its default wildcards, see here: 
https://forums.zotero.org/discussion/82766/renaming-rules-not-taken-into-account

BTW, to set up user-defined wildcards, it's best to first have a look at `wildcards.default` for examples. Some of the examples on the ZotFile website are outdated.

If you want to rename files automatically or with a keyboard shortcut, see [here](https://forums.zotero.org/discussion/comment/357133/#Comment_357133).
